packages/service-clients/end-to-end-tests/azure-client/package.json

```diff
@@ -31,11 +31,12 @@
 		"test": "npm run test:realsvc",
 		"test:coverage": "c8 npm test",
 		"test:realsvc": "npm run test:realsvc:tinylicious",
-		"test:realsvc:azure": "cross-env FLUID_CLIENT=azure npm run test:realsvc:azure:run",
+		"test:realsvc:azure": "cross-env FLUID_CLIENT=azure npm run test:realsvc:azure:run -- --driver=r11s --r11sEndpointName=frs",
 		"test:realsvc:azure:run": "mocha --recursive \"lib/test/**/*.spec.*js\" --exit --timeout 20000 --config src/test/.mocharc.cjs",
 		"test:realsvc:run": "mocha lib/test --config src/test/.mocharc.cjs",
-		"test:realsvc:tinylicious": "start-server-and-test start:tinylicious:test 7071 test:realsvc:azure:run",
+		"test:realsvc:tinylicious": "start-server-and-test start:tinylicious:test 7071 test:realsvc:tinylicious:run",
```

Comment:
was this just a mistake before?

Reply:
`test:realsvc:tinylicious:run` does run `test:realsvc:azure:run` as before but I have added a new script for clarity. we do have a similar script in end-to-end tests: https://github.com/microsoft/FluidFramework/blob/main/packages/test/test-end-to-end-tests/package.json#L57